apps/ledger-live-mobile/package.json

```diff
@@ -140,14 +140,14 @@
     "react-native-adjust": "^4.29.6",
     "react-native-android-location-services-dialog-box": "^2.8.2",
     "react-native-animatable": "^1.3.3",
+    "react-native-biometrics": "^3.0.1",
     "react-native-ble-plx": "2.0.3",
     "react-native-codegen": "^0.0.7",
     "react-native-config": "1.4.4",
     "react-native-easy-markdown": "^2.0.0",
     "react-native-extra-dimensions-android": "^1.2.5",
     "react-native-fast-crypto": "^2.2.0",
     "react-native-fast-image": "^8.5.11",
-    "react-native-fingerprint-scanner": "git+https://github.com/hieuvp/react-native-fingerprint-scanner.git#f1d136f605412d58e4de9e7e155d6f818ba24731",
```

Comment:
Thank god we remove this atrocity XD

Reply:
Just a bit sad that we traded 1 unmaintained library for another :( 
Last publish: 2 years ago 😢

Reply:
Actually not https://github.com/SelfLender/react-native-biometrics latest release is from 2 month ago